sql/ts/tests/node_modules

```diff
@@ -0,0 +1 @@
+../../../node_modules/
```

Comment:
o_O

Reply:
This is a symlink because the npm workspaces feature causes packages to be installed at the workspace root. It's a bit dirty but skdb seems to rely explicitly on some `node_modules` paths.